sparql/src/test/java/org/apache/zeppelin/sparql/SparqlJenaEngineTest.java

```diff
@@ -108,6 +108,10 @@ void testSuccessfulRawQuery() {
     assertEquals(Code.SUCCESS, result.code());
 
     final String expected = "?subject\t?predicate\t?object\n" +
```

Comment:
Apparently the sorting has been changed a little. I hope it doesn't cause any problems.